ansible_collections/arista/avd/roles/eos_validate_state/tasks/router_id_reachability.yml

```diff
@@ -0,0 +1,56 @@
+---
+
+- name: Gather ip reachability state between devices (Router ID <-> Router ID)
```

Comment:
IMO we should call this `loopback_reachability`, since router-id is not strictly an IP, but just a 32-bit number.
We could pick up the first loopback interface in `loopback_interfaces` as long as it is _not_ the VTEP.
If we need to add another variable to structured_config, we can do this - we just need to document it and comment that it is only used for validation role. (and maybe documentation?)

Reply:
Since this test is very specific for EVPN, what if we take the ``update_source`` variable from ``EVPN-OVERLAY-PEERS``? This makes the test non-reusable for L2LS or L3LS but as said, I think it is not needed.